mesh/pmesh.hpp

```diff
@@ -496,10 +526,6 @@ class ParMesh : public Mesh
    void GenerateOffsets(int N, HYPRE_BigInt loc_sizes[],
                         Array<HYPRE_BigInt> *offsets[]) const;
 
-   /** Return true if the face is interior or shared. In parallel, this
-       method only works if the face neighbor data is exchanged. */
-   inline bool FaceIsTrueInterior(int FaceNo) const { return Mesh::FaceIsTrueInterior(FaceNo); }
```

Comment:
Can this be kept for backwards compatibility?

Reply:
The `using Mesh::FaceIsTrueInterior` just changes the access protection for the base method, so is equivalent to this forwarding dispatch.